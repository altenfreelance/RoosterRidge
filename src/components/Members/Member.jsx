import React, { Component } from "react";
import "./member.css";

export default class Member extends Component {
  render() {
    let { name, instrument, img_url } = this.props;
    return (
      <div className="col-sm-4 p-3">
        <div className="card">
          <div className="container">
            <h4>
              <b>{name}</b>
            </h4>
            <p>{instrument}</p>
          </div>
          <img src={img_url} alt={name} className="rounded-circle" style={{ width: "100%" }} />
          </div>
      </div>
    );
  }
}
