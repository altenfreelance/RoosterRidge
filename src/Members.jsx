import React from 'react'
import Member from './Member'
// import andrew_img from 'images/Andrew.jpg'
// import jacob_img from './images/Jacob.jpg'
// import bryce_img from './images/Bryce.jpg'
// import greg_img from './images/Greg.jpg'
// import joey_img from './images/Joey.jpg'
// import vince_img from './images/Vince.jpg'
// import blum_img from './images/Dave.jpg'
// import bauer_img from './images/OLAS.jpg'
function Members(){
    const andrew_img = 'images/Andrew.jpg';
    const jacob_img = 'images/Jacob.jpg'
    const bryce_img = 'images/Bryce.jpg'
    const greg_img = 'images/Greg.jpg'
    const joey_img = 'images/Joey.jpg'
    const vince_img = 'images/Vince.jpg'
    const blum_img = 'images/Dave.jpg'
    const bauer_img = 'images/OLAS.jpg'

        return (
            <div>
                <h1>Meet the Band</h1>
                <div className="row">
                    <Member  name="Andrew Alten" instrument="Mandolin" img_url={andrew_img}/>
                    <Member  name="Jacob Alten" instrument="Vocal and Guitar" img_url={jacob_img}/>
                    <Member  name="Bryce Clawson" instrument="Vocals and Harmonica " img_url={bryce_img}/>
                    <Member  name="Greg Stevens" instrument="Banjo and Guitar" img_url={greg_img}/>
                    <Member  name="Joey Oberholzer" instrument="Vocals" img_url={joey_img}/>
                    <Member  name="Vince Stevens" instrument="Dobro" img_url={vince_img}/>
                    <Member  name="Dave Blumberg" instrument="Bass" img_url={blum_img}/>
                    <Member  name="Dave Bauer" instrument="Sound" img_url={bauer_img}/>
                </div>
            </div>
        )
    
}

export default Members;
