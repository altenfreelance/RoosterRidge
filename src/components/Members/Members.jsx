import React from 'react'
import Member from './Member'

function Members(){
    const andrew_img = 'images/members/Andrew.jpg';
    const jacob_img = 'images/members/Jacob.jpg'
    const bryce_img = 'images/members/Bryce.jpg'
    const greg_img = 'images/members/Greg.jpg'
    const joey_img = 'images/members/Joey.jpg'
    const vince_img = 'images/members/Vince.jpg'
    const jamie_img = 'images/members/Jamie.jpg'
    const bauer_img = 'images/members/OLAS.jpg'

        return (
            <div>
                <h1>Meet the Band</h1>
                <div className="row">
                    <Member name="Andrew Alten" instrument="Mandolin" img_url={andrew_img}/>
                    <Member name="Jacob Alten" instrument="Vocal and Guitar" img_url={jacob_img}/>
                    <Member name="Bryce Clawson" instrument="Vocals and Harmonica " img_url={bryce_img}/>
                    <Member name="Greg Stevens" instrument="Banjo and Guitar" img_url={greg_img}/>
                    <Member name="Joey Oberholzer" instrument="Vocals" img_url={joey_img}/>
                    <Member name="Vince Stevens" instrument="Dobro" img_url={vince_img}/>
                    <Member name="Jamie" instrument="Bass" img_url={jamie_img}/>
                    <Member name="Dave Bauer" instrument="Sound" img_url={bauer_img}/>
                </div>
            </div>
        )
    
}

export default Members;
