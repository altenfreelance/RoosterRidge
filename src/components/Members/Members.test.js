import React from 'react';
import Members from './Members';

import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';

afterEach(cleanup);

it('renders (matches snapshot) ', () => {
    const {asFragment} = render(<Members />);
    expect(asFragment()).toMatchSnapshot();
});
