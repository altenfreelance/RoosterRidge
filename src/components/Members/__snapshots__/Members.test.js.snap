// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders (matches snapshot)  1`] = `
<DocumentFragment>
  <div>
    <h1>
      Meet the Band
    </h1>
    <div
      class="row"
    >
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Andrew Alten
              </b>
            </h4>
            <p>
              Mandolin
            </p>
          </div>
          <img
            alt="Andrew Alten"
            class="rounded-circle"
            src="images/members/Andrew.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Jacob Alten
              </b>
            </h4>
            <p>
              Vocal and Guitar
            </p>
          </div>
          <img
            alt="Jacob Alten"
            class="rounded-circle"
            src="images/members/Jacob.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Bryce Clawson
              </b>
            </h4>
            <p>
              Vocals and Harmonica 
            </p>
          </div>
          <img
            alt="Bryce Clawson"
            class="rounded-circle"
            src="images/members/Bryce.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Greg Stevens
              </b>
            </h4>
            <p>
              Banjo and Guitar
            </p>
          </div>
          <img
            alt="Greg Stevens"
            class="rounded-circle"
            src="images/members/Greg.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Joey Oberholzer
              </b>
            </h4>
            <p>
              Vocals
            </p>
          </div>
          <img
            alt="Joey Oberholzer"
            class="rounded-circle"
            src="images/members/Joey.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Vince Stevens
              </b>
            </h4>
            <p>
              Dobro
            </p>
          </div>
          <img
            alt="Vince Stevens"
            class="rounded-circle"
            src="images/members/Vince.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Jamie Davidson
              </b>
            </h4>
            <p>
              Bass
            </p>
          </div>
          <img
            alt="Jamie Davidson"
            class="rounded-circle"
            src="images/members/Jamie.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
      <div
        class="col-sm-4 p-3"
      >
        <div
          class=""
        >
          <div
            class="container"
          >
            <h4>
              <b>
                Dave Bauer
              </b>
            </h4>
            <p>
              Sound
            </p>
          </div>
          <img
            alt="Dave Bauer"
            class="rounded-circle"
            src="images/members/OLAS.jpg"
            style="width: 100%;"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
